cmd/kops/validate_cluster.go

```diff
@@ -172,8 +172,8 @@ func RunValidateCluster(f *util.Factory, cmd *cobra.Command, args []string, out
 		// do we need to print which instance group is not ready?
 		// nodes are going to be a pain
 		fmt.Fprint(out, "\nValidation Failed\n")
-		fmt.Fprintf(out, "Master(s) Not Ready %d out of %d.\n", len(validationCluster.MastersNotReadyArray), validationCluster.MastersCount)
-		fmt.Fprintf(out, "Node(s) Not Ready   %d out of %d.\n", len(validationCluster.NodesNotReadyArray), validationCluster.NodesCount)
+		fmt.Fprintf(out, "Ready Master(s) %d out of %d.\n", len(validationCluster.MastersNotReadyArray), validationCluster.MastersCount)
```

Comment:
I like this message.  Is this the wrong way round though?  If one master is not ready, and 4 are ready, it looks like this prints:

"Ready Masters 1 out of 5"